AmplifyPlugins/DataStore/Podfile.lock

```diff
@@ -22,16 +22,16 @@ PODS:
     - AWSPluginsCore (= 1.1.1)
   - AppSyncRealTimeClient (1.4.0):
     - Starscream (~> 3.1.0)
-  - AWSAuthCore (2.15.1):
```

Comment:
Make sure to update all other Podfile.lock files also. Alternately, remove this and approve https://github.com/aws-amplify/amplify-ios/pull/743, then rebase against `main` :)

Reply:
Updated